apps/frontend/src/components/global/ExportCKLModal.vue

```diff
@@ -526,6 +555,52 @@ export default class ExportCKLModal extends Vue {
     return macPattern.test(value) || 'Invalid MAC Address Format';
   }
 
+  setProperName(name: string, fileIndex: number, profileIndex: number): string {
+    let newName = name;
+
+    // Find if we need to format the name
+    let index = 0;
+    const baselineArray = ['stig-baseline', 'cis-baseline', 'srg-baseline'];
+    for (const baseline of baselineArray) {
+      if (name.indexOf(baseline) > 0) {
```

Comment:
I know this works for every relevant use case, but is this to test whether a string in the baselineArray is present in the name then should it not be '> -1'? Otherwise this excludes names where a baselineArray value is at the beginning of the string.